### DISCUSSION TITLE
[UI] Add custom column to topic list page

### DISCUSSION THREAD

**Discussion by bartektartanus** [tchiotludo/akhq#1067] [General]
In our system, we have a dashboard in Grafana for each Kafka topic. Would be nice to have a button that redirects users to that dashboard. 
Is this possible with inject-js-code option?
https://akhq.io/docs/configuration/akhq.html#inject-some-css-or-javascript

**Comment by tchiotludo** [comment]
`akhq.html-head` allow you to inject any css or javascript inside the page.
Maybe you can watch the current url and add a button on the interface ? 
Since it's react with infamous div name I think it's the simplest solution.
